distance_tolerance: 0.7

base_frame_id: base_link
camera:
  frame_id: camera_center
  x_min: 1.7
  x_max: 8.0
lidar:
  frame_id: lidar
  x_min: 0.75
  x_max: 8.0

# data export
data_filename: fusion_data
map_frame: odom
